python/Ganga/GPIDev/Schema/Schema.py

```diff
@@ -402,7 +402,12 @@ def __len__(self):
     # item.isA(SimpleItem)
     def isA(self, _what):
 
-        from Ganga.GPIDev.Base.Proxy import stripProxy
+        try:
+            from Ganga.GPIDev.Base.Proxy import stripProxy
+        except ImportError:
+            def dummyfunc(arg):
+                return arg
+            stripProxy = dummyfunc
```

Comment:
In that case this can be simplified by removing the stripProxy entirely but the ```isA``` is used in many places in the source code so I'm weary of this suddenly causing things to re-break again